cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -144,3 +165,176 @@ def validate(self) -> list[DMSSchemaError]:
                     errors.add(DuplicatedViewInDataModelError(referred_by=model.as_id(), view=view_id))
 
         return list(errors)
+
+
+@dataclass
+class PipelineSchema(DMSSchema):
```

Comment:
Do we refer in CDF to this as `Pileline` or as `Transformations` ? If the latter, I would strongly suggest sticking to the CDF naming convention

Reply:
Pipeline = Raw table + transformations